platform_channels/lib/src/image_basic_message_channel.dart

```diff
@@ -0,0 +1,24 @@
+import 'dart:typed_data';
```

Comment:
nit: Copyright header needed.

Reply:
I believe there should be a whitespace line between these two imports, since one comes from the Dart SDK and one is from a package.